runtime/src/iree/tools/utils/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 ################################################################################
 # Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
-# iree/tools/utils/BUILD                                                       #
+# runtime/src/iree/tools/utils/BUILD                                           #
```

Comment:
The `cmake_baremetal_riscv32` build is currently relying on this early exit turning off the `yaml` target (used via the `yaml_util` and `trace_replay` libraries):

https://github.com/google/iree/blob/500e41138cfa47b535de33a34c5d3de9eae25662/iree/tools/CMakeLists.txt#L9-L13

[error logs here](https://source.cloud.google.com/results/invocations/88920622-5240-4063-9360-e5e325a39090/targets/iree%2Fgcp_ubuntu%2Fcmake%2Fbaremetal%2Friscv32%2Fpresubmit/log):
```c
../third_party/libyaml/src/api.c:66:27: error: implicit declaration of function 'strdup' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    return (yaml_char_t *)strdup((char *)str);
                          ^
../third_party/libyaml/src/api.c:66:27: note: did you mean 'strcmp'?
/usr/src/toolchain_iree_rv32imf/bin/../riscv32-unknown-elf/include/string.h:36:6: note: 'strcmp' declared here
int      strcmp (const char *, const char *);
         ^
```